CODE_REVIEW.md

```diff
@@ -0,0 +1,47 @@
+**Are there any problems or code smells in the app?**
+
+    1. Test case fails When running the code:
+
+        - Fixed the failed test case in the READING-LIST.REDUCER.SPECS.TS for missed out the reducers in reading-list.reducer.ts
+
+        - Added test cases for the add book to readlist and remove book from readlist in the file READING-LIST.REDUCER.SPECS.TS.
+
+    2.  Without unsubscribe causes Memory leak which can handle by using ```async``` instead of subsrciption which handles the memory leaks by its own.
+
+    3. Missed accessibilities are added and its passed 100% using lighthouse.
+
+    4. Types not properly defined, use relevant types or void if function not return anything, For instance reading list component.
+
+    5. In Book Search component instead of writing seperate dateformat we can format by using filter pipe in html itself which is implemented.
+
+    6. All actions names can be export by single enum constant.
```

Comment:
We shouldn't do this since it's unnecessary boilerplate. Action creators already give you access to its type so I would keep it inlined in there.

```
const myAction = createAction('[My UI]');
console.log(myAction.type); // 'My UI'
```